template/studio/schemas/author.js

```diff
@@ -32,21 +29,14 @@ export default {
     },
     {
       name: 'bio',
-      title: 'Bio',
-      type: 'array',
-      of: [
-        {
-          title: 'Block',
-          type: 'block',
-          styles: [{title: 'Normal', value: 'normal'}],
-          lists: []
-        }
-      ]
+      type: 'simplePortableText',
```

Comment:
I think we should create unique portable text types for each place we use that. E.g. `bioPortableText` in this case.

Reply:
Why?